src/Http/Controllers/Configuration/UserController.php

```diff
@@ -75,17 +79,14 @@ public function editUser($user_id)
      * @param \Seat\Web\Http\Validation\EditUser $request
      *
      * @return mixed
+     * @throws \Seat\Services\Exceptions\SettingException
      */
     public function updateUser(EditUser $request)
     {
 
         $user = $this->getUser($request->input('user_id'));
 
-        $user->fill([
-            'email' => $request->input('email'),
-        ]);
-
-        $user->save();
+        Profile::set('email_address', $request->input('email'), $user->group->id);
```

Comment:
Please use the helper instead making direct call to this class : `setting(['email_addres', $request->input('email'), $user->group->id]);`

https://github.com/eveseat/services/blob/master/src/Helpers/helpers.php#L267